cdc/sink/mysql/mysql.go

```diff
@@ -233,6 +229,7 @@ func (s *mysqlSink) EmitRowChangedEvents(ctx context.Context, rows ...*model.Row
 
 // FlushRowChangedEvents will flush all received events, we don't allow mysql
 // sink to receive events before resolving
+// Concurrency Note: FlushRowChangedEvents is thread-safe.
 func (s *mysqlSink) FlushRowChangedEvents(ctx context.Context, tableID model.TableID, resolvedTs uint64) (uint64, error) {
 	// Since CDC does not guarantee exactly once semantic, it won't cause any problem
```

Comment:
This comment here can be remove as that now we will delete `tableID` when a table is removed.